packages/components/src/button/style.scss

```diff
@@ -194,7 +194,41 @@
 
 	// Link buttons that are red to indicate destructive behavior.
 	&.is-link.is-destructive {
-		color: $alert-red;
+		color: #b52727;
+	}
+
+	/**
+	 * 	Destructive buttons.
+	 */
+
+	&.is-destructive {
+		color: #b52727;
+		box-shadow: inset 0 0 0 $border-width #b52727;
+
+		&:hover:not(:disabled) {
+			color: #a02222;
```

Comment:
This is a darker version of your red:

![Rectangle 201](https://user-images.githubusercontent.com/1204802/83874421-b4675e80-a735-11ea-906f-7605f104a97c.png)

It's a good call for the hover, however we should probably use some SCSS smarts so we can reuse the variable. You could, for example, use `color: darken($alert-red, 20%);`.